parser/ndt.go

```diff
@@ -9,25 +9,23 @@ import (
 	"strings"
 
 	"cloud.google.com/go/bigquery"
+	"github.com/prometheus/client_golang/prometheus"
+
 	"github.com/m-lab/etl/bq"
+	"github.com/m-lab/etl/etl"
 	"github.com/m-lab/etl/metrics"
 	"github.com/m-lab/etl/web100"
-
-	"github.com/m-lab/etl/etl"
-)
-
-const (
-	// TODO(prod): eliminate need for tmpfs.
-	tmpDir = "/mnt/tmpfs"
 )
 
 type NDTParser struct {
-	inserter etl.Inserter
-	tmpDir   string
+	inserter  etl.Inserter
+	tableName string
+	// TODO(prod): eliminate need for tmpfs.
+	tmpDir string
 }
 
-func NewNDTParser(ins etl.Inserter) *NDTParser {
-	return &NDTParser{ins, tmpDir}
+func NewNDTParser(ins etl.Inserter, tableName string) *NDTParser {
```

Comment:
I removed the `tableName` parameter to be more like the `NewDiscoParser` so that the tablename comes from the inserter. Was this correct?